interview.md

```diff
@@ -58,11 +121,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+About 50-60% of the time would be spent engaging with the language aspects, while 40-50% will be working on the systems aspects of this project. There are equally as many concerns about how much natural language should be incorporated into the DSL as how many customizations should users be able to define and assumptions that the program should make.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think the DSL for this idea is a well-scoped idea for the class, and the inputs and outputs are relatively easy to parse and generate. Since this need is, at its root, a text-processing problem, many parts of the project can be reduced down to strings to simplify the implementation.
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+The benefits of this project include familiarity with the domain. The problem is also well-defined, and constraints can also be easily transferred into code. The project also has the flexibility of not adding too much optimization between constraints as the goal is to have loosely-ordered documents. The drawbacks of this project include dealing with multiple data types in the “backend” of the DSL without revealing it to the user. This limited information may cause some unintended issues and edge cases that are difficult to understand at the moment.
```

Comment:
I think that this is a suitable project topic that would allow you to get hands on experience creating a language.

Out of all three of your topics, this one seems the most doable and suited for this class